src/requireCssModule.js

```diff
@@ -18,19 +18,57 @@ import type {
   StyleModuleMapType
 } from './types';
 
-const getTokens = (runner, cssSourceFilePath: string, filetypes): StyleModuleMapType => {
+type FileTypeOptions = {|
+  syntax: string,
+  plugins: Array<string>
+|};
+
+const getFiletypeOptions = (cssSourceFilePath: string, filetypes: Object): ?(string|FileTypeOptions) => {
```

Comment:
`?(string|FileTypeOptions)` does not match the return type (not getting an error here?)  `FileTypeOptions | string | null`.

Where does the `string` come from, though?